### ISSUE TITLE
integrate "BrowserID and CSP" instructions into an official documentation page

### ISSUE THREAD

**Issue by floatingatoll** [mozilla/persona#1665]
http://feeding.cloud.geek.nz/2011/11/using-browserid-and-content-security.html

These are terrifically useful instructions for setting up BrowserID and CSP together. Can we merge them into an official documentation page somewhere? Rumor has it the author works on BrowserID :)

**Comment by callahad** [comment]
Will do!

**Comment by lloyd** [comment]
+1.  I hear the author of that is a pretty solid dude.

**Comment by seanmonstar** [comment]
Did this happen?

**Comment by callahad** [comment]
Not yet; was blocked on MDN. Now that MDN staging is editable, I'll add this back to my TODO queue. Thanks for pinging me!

**Comment by callahad** [comment]
CSP docs folded into https://developer.mozilla.org/en-US/docs/Persona/Security_Considerations
